manager.py

```diff
@@ -329,8 +329,8 @@ def run(self):
                     self.vprint(1, 'Reboot: taking down Manager')
                     self.stop()
                     self.takedown()
-                    os.system('sudo {0} {1}'.format(
-                        REBOOT_SCRIPT, self.branch))
+                    #os.system('sudo {0} {1}'.format(
+                        #REBOOT_SCRIPT, self.branch))
```

Comment:
Actually, I wouldn't change/remove this command until we have another way of initiating the reboot. Otherwise, we'll break the ability to initiate an update remotely.